pilot/pkg/model/telemetry.go

```diff
@@ -116,10 +124,33 @@ func shallowMerge(parent, child *tpb.Telemetry) *tpb.Telemetry {
 	}
 	merged := parent.DeepCopy()
 	shallowMergeTracing(merged, child)
+	shallowMergeMetrics(merged, child)
 	shallowMergeAccessLogs(merged, child)
 	return merged
 }
 
+func shallowMergeMetrics(parent *tpb.Telemetry, child *tpb.Telemetry) {
```

Comment:
nit: should we call this `mergeMetrics` in anticipation of the more nuanced merging that will happen?